projects/samples/robotbenchmark/highway_driving/plugins/robot_windows/highway_driving/highway_driving.html

```diff
@@ -2,59 +2,79 @@
+          </p>
+        </div>
+        <input class="radio" id="instructions" name="group" type="radio">
+        <label class="tab" for="instructions">Instructions</label>
+        <div class="panel">
+          <p>
+            <h2>How to improve the performance?</h2>
+          </p>
+          <p>
+            The basic controller is a simple program that makes the vehicle go straight and decrease its speed if another vehicle is detected in front in order to keep a security distance.
+          </p>
+          <p>
+            Instead of decreasing the speed, the lateral <a target=_blank href="https://www.cyberbotics.com/doc/reference/distancesensor">DistanceSensor</a> devices can be used to check if it is possible to change lane in order to overtake (you are allowed to overtake by the right side).
+            Here are the available <a target=_blank href="https://www.cyberbotics.com/doc/reference/distancesensor">DistanceSensor</a> devices:
+            <div align=center>
+              <img width=500 src='sensors.png'>
+            </div>
+          </p>
+          <p>
+            Note that after a while the road becomes curvy, you will therefore need to take the direction of the road into account.
+            A simple way to do so is to use the lateral <a target=_blank href="https://www.cyberbotics.com/doc/reference/distancesensor">DistanceSensors</a> in order to know the distance to the right and left barriers.
+            A more elegant way would be to use the <a target=_blank href="https://www.cyberbotics.com/doc/reference/camera">Camera</a> in order to detect the road lines.
+          </p>
+          <p>
+            In addition to the <a target=_blank href="https://www.cyberbotics.com/doc/reference/distancesensor">DistanceSensors</a>, the vehicle is equipped with a <a target=_blank href="https://www.cyberbotics.com/doc/reference/gps">GPS</a> located in the center of the rear-wheels axes,
+            and a frontal <a target=_blank href="https://www.cyberbotics.com/doc/reference/camera">Camera</a> (with objects <a target=_blank href="https://www.cyberbotics.com/doc/reference/recognition">Recognition</a> capability).
+          </p>
+        </div>
       </div>
     </div>
     <script src="highway_driving.js"></script>
+    <script>
+      let height;
+      function adaptPositionOfText() {
+        let tabs = document.getElementsByClassName('tab')
+        if(!tabs.length > 0)
+          return
+        let newHeight = tabs[tabs.length - 1].getBoundingClientRect().top + tabs[tabs.length - 1].getBoundingClientRect().height;
+        if(newHeight === height)
+          return;
+        height = newHeight;
+        let panels = document.getElementsByClassName('panel');
+        for (panel of panels) {
+          panel.style.top = height + 'px';
+        };
+      }
+      window.addEventListener('resize', adaptPositionOfText);
+      adaptPositionOfText();
+    </script>
```

Comment:
Just a question: why this was not needed before when using ids instead of classes in the CSS?
Shouldn't work exactly the same?

Reply:
Because with ids we had a structure like:
```
<input></input>
<input></input>
<input></input>

<label class=tabs></label>
<label class=tabs></label>
<label class=tabs></label>

<div class=panel></panel>
<div class=panel></panel>
<div class=panel></panel>
```
Because we were using IDs, it was easy to connect one input to one particular div.
And also the element were displayed in order: the tabs on top, the panel below.

However when switching for classes the structures changed like this:
```
<input></input>
<label class=tabs></label>
<div class=panel></panel>

<input></input>
<label class=tabs></label>
<div class=panel></panel>

<input></input>
<label class=tabs></label>
<div class=panel></panel>
```

It is the case because as we don't have IDs anymore, it is way harder to connect one specific input to a specific panel. The simplest solution with only CSS I found was to connect the input to the nearest panel below it. But as you can see the structure is not the same without further changes it would display like:

```
<Overview>
text_of_panel, text_of_panel, text_of_panel, text_of_panel, text_of_panel, text_of_panel
text_of_panel, text_of_panel, text_of_panel, text_of_panel, text_of_panel, text_of_panel, 
text_of_panel, text_of_panel, text_of_panel, text_of_panel, text_of_panel, text_of_panel
<Metric>
<Instructions>
```

To solved that I made the position of the panels absolute. Problem: if we resize the window and the buttons are placed on several columns, we need to update the position of the panels to always be just below the lowest button. And that what the JS code does. 

I am not sure this is an optimal solution, but it is the simplest I could come up with without IDs and then scalable.